blog/releases/announcing-knative-v0-22-release.md

```diff
@@ -0,0 +1,195 @@
+- Fix the incorrect Gateway name format for DomainMapping auto TLS feature for net-istio implmenetation. [net-istio#532](https://github.com/knative-sandbox/net-istio/pull/532)
+
+
+### Eventing v0.22
+
+#### 🚨 API Changes
+
+- v1alpha1 Channel duck types are no longer supported. [#5005](https://github.com/knative/eventing/pull/5005)
+- Allow Trigger users to set another namespace's Subscriber, And when Subscriber's namespace not been set, it will be set to Trigger's namespace [#4995](https://github.com/knative/eventing/pull/4995)
+
+#### 🐞 Bug Fixes
+
+- Fix PingSource adapter not always releasing the leader election lease when terminating [#5162](https://github.com/knative/eventing/pull/5162)
+- Fix PingSource sending duplicate events when the leading adapter fails to renew its lease [#4908](https://github.com/knative/eventing/pull/4908)
+- Fix bug preventing the namespace-scoped in-memory channel to become ready. [#4906](https://github.com/knative/eventing/pull/)
+- Fix excessive number of Replicas for eventing-webhook [#5112](https://github.com/knative/eventing/pull/5112)
+- Webhook timeout for API server calls has been increased to 10 seconds. [#5175](https://github.com/knative/eventing/pull/5175)
+
+#### 🧹 Clean up
+
+- Do not set the finalizer for pingSource for pingSource controller, just deal with it in reconcilekind [#5002](https://github.com/knative/eventing/pull/5002)
+
+### Eventing Extensions
+
+
+#### Eventing Kafka Broker v0.22
+
+#### 🚨 Breaking or Notable
+
+- Kubernetes's minimum version is 1.18. [#779](https://github.com/knative-sandbox/eventing-kafka-broker/pull/779)
+
+#### 💫 New Features & Changes
+
+- Support ordered delivery.
+  - You can choose between ordered and unordered delivery through the label kafka.eventing.knative.dev/delivery.order in your triggers.
+  - Check out the docs for more details at https://knative.dev/docs/eventing/broker/kafka-broker . [#589](https://github.com/knative-sandbox/eventing-kafka-broker/pull/)
+
+#### 🐞 Bug Fixes
+
+- Add producer interceptor io.cloudevents.kafka.PartitionKeyExtensionInterceptor to provide ordered delivery based on the partitioning extension of the CloudEvents spec. [#751](https://github.com/knative-sandbox/eventing-kafka-broker/pull/751)
+- Fix unable to deploy KafkaSink without Kafka Broker installed [#714](https://github.com/knative-sandbox/eventing-kafka-broker/pull/714)
+
+### Client v0.22
+
+kn 0.22.0 comes with some bug fixes and minor feature enhancements. It's mostly a polishing release. If you are using the client API, there is a breaking change that was needed to align with Kubernetes client API's
+
+#### META
+The compile dependencies have been updated to Knative Serving 0.22.0 and Knative Eventing 0.22.0. There are no changes in the used API version for communicating with the backend.
```

Comment:
Move this to earlier in the doc, also the META heading is not clear. I don't think this needs a whole heading, could probably just go in the highlights section.

Reply:
I agree I removed it from the doc, is more internal info